.docker/src-waf/etc/nginx/snippets/domain-rewrites-benefits-tool.conf

```diff
@@ -0,0 +1,11 @@
+    # Rewrites for requests for Host: benefits-tool.usa.gov
+
+    rewrite ^/death-of-a-loved-one https://www.usa.gov/benefit-finder/death permanent;
+    rewrite ^/disability https://www.usa.gov/benefit-finder/disability permanent;
+    rewrite ^/retirement https://www.usa.gov/benefit-finder/retirement permanent;
```

Comment:
In the google doc this is written as going to usa.gov/benefit-finder/disability as well. I am assuming it's a mistake in the google doc since I see in Spanish retirement and disability have two separate urls but I just want to make sure

Reply:
Hey, they just updated the google doc and I didn't even have to ask!